src/components/UI/card/Card.css

```diff
@@ -3,31 +3,35 @@
 .card {
   composes: base container rounded depth;
   overflow: hidden;
-}
-
-.card > *:not(iframe, video, img, header, footer) {
-  margin-right: 10px;
-  margin-left: 10px;
-}
-
-.card > *:not(iframe, video, img, header, footer):first-child {
-  margin-top: 10px;
-}
-
-.card > *:not(iframe, video, img, header, footer):last-child {
-  margin-bottom: 10px;
+  border: 1px solid #f7f8f8;
+  transition: all .1s ease-in-out;
+  transform: translateY(0);
+  padding: 16px 24px;
 }
 
 .card > iframe,
 .card > video,
 .card > img {
-  max-width: 100%;
+  margin: -16px -24px 16px -24px;
+  width: calc(100% + 16px + 16px);
 }
 
 .card h1 {
-  margin: 15px 0;
+  font-size: 16px;
+  font-weight: 600;
+  letter-spacing: 0;
+  line-height: 24px;
+  margin: 0;
   padding: 0;
   border: none;
   color: var(--defaultColor);
-  font-size: 18px;
+}
+
+.card p {
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: 24px;
+  padding: 0;
+  color: #8c8c8c;
```

Comment:
I don't think this should be hardcoded - we should add it as a text color in `theme.css`.